Tests/DependencyInjection/AbstractDoctrineExtensionTest.php

```diff
@@ -856,13 +856,19 @@ public function testAttachEntityListenerTag()
         $this->compileContainer($container);
 
         $listener = $container->getDefinition('doctrine.orm.em1_entity_listener_resolver');
-        $this->assertDICDefinitionMethodCallOnce($listener, 'registerService', ['EntityListener1', 'entity_listener1']);
+        $this->assertDICDefinitionMethodCallCount($listener, 'registerService', [
+            ['EntityListener1', 'entity_listener1'],
+            ['Doctrine\Bundle\DoctrineBundle\Tests\DependencyInjection\Fixtures\InvokableEntityListener', 'invokable_entity_listener'],
+            ['Doctrine\Bundle\DoctrineBundle\Tests\DependencyInjection\Fixtures\InvokableEntityListener', 'invokable_entity_listener'],
+        ], 3);
```

Comment:
I think tagging mutiple times the same listener should not register the service multiple times. We could avoid it in the `EntityListenerPass`. WDYT ?

Reply:
If it's the same service and not registering it twice doesn't cause any issues, I'm all for it 👍

Reply:
I will take a look at it but in another PR because this change is unrelated to the proposed new feature. Is that alright for you?

Reply:
Yes, that’s alright 👍